vertx-db2-client/src/main/java/io/vertx/db2client/impl/drda/DRDAQueryRequest.java

```diff
@@ -1399,7 +1409,7 @@ private Hashtable computeProtocolTypesAndLengths(
                     lidAndLengths[i][0]--;
                 }
             }
-            return overrideMap;
+            return overrideMap; // @AGG this is never used
```

Comment:
Intentionally leaving this comment here so I remember to clean it up once I do the global cleanup of the drda package

Reply:
Thanks for letting me know